src/main/java/com/odysseusinc/arachne/portal/api/v1/controller/BaseAnalysisController.java

```diff
@@ -371,7 +374,7 @@ public JsonResult updateCommonEntityInAnalysis(@PathVariable("analysisId") Long
     @RequestMapping(value = "/api/v1/analysis-management/analyses/{analysisId}/upload", method = POST)
     public JsonResult<List<AnalysisFileDTO>> uploadFile(Principal principal,
                                                         @RequestParam(required = false) MultipartFile[] file,
-                                                        @RequestParam String label,
+                                                        @Size(min = 1, message = "Label cannot be empty") @RequestParam String label,
```

Comment:
It is'n best idea to place validation here, i think it would be better to move it out to dto